packages/block-library/src/navigation-menu-item/edit.js

```diff
@@ -0,0 +1,250 @@
+/**
+ * External dependencies
+ */
+import { invoke } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import {
+	Dropdown,
+	ExternalLink,
+	IconButton,
+	MenuItem,
+	NavigableMenu,
+	PanelBody,
+	TextareaControl,
+	TextControl,
+	ToggleControl,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import {
+	InspectorControls,
+	PlainText,
+} from '@wordpress/block-editor';
+import { withDispatch } from '@wordpress/data';
+import { compose } from '@wordpress/compose';
+import {
+	Component,
+	createRef,
+	Fragment,
+} from '@wordpress/element';
+
+const MenuItemActions = compose( [
```

Comment:
This file has already 250 lines. I'm wondering if `MenuItemActions` should be extracted to its file.